dev/components/app-calendar-content.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LitElement, html} from 'lit';
+
+import './app-calendar-body';
+import './app-calendar-content-header';
+
+/**
+ * `<app-calendar-content>` Custom component to add a calendar content
+ *
+ * <app-calendar>
+ *  <app-calendar-content></app-calendar-content>
+ *
+ * @polymer
+ * @litElement
+ * @customElement
+ */
+export class AppCalendarContent extends LitElement {
+
+  /**
+   * Static getter properties
+   * 
+   * @returns Object
+   */
+  static get properties() {
+    return {
+      /**
+       * holds the current date to represent the month to pass down to its children components
+       */
+      currentMonth: {type: Object},
+
+      /**
+       * holds the current date to pass down to its children components
+       */
+      selectedDate: {type:Object},
+
+      /**
+       * array to hold event passed from parent and pass it down
+       */
+      events: {type:Array},
+
+      /**
+       * handler function that triggers when user click on more events option
+       */
+      onMoreMenuClick: {type: Function},
+
+      /**
+       * handler function when event changes on drag
+       */
```

Comment:
initialize all these props in  a constructor. Or else initially all the values of the props will be set to undefined

Reply:
since all these props are not used by this component it self and only passes them to children. does it still need to be initialized?

Reply:
yes..whenever you define a props its a good approach to initialize it in the constructor to avoid possible anomalies.